### ISSUE TITLE
Cannot able to see meta information in auto completion box.

### ISSUE THREAD

**Issue by yashvekaria** [ajaxorg/ace#3681]
As you can see from live [demo](https://ace.c9.io/build/kitchen-sink.html) kitchen sink example. You can clearly able to view whether entered word is of type "snippet" or "local".

![3](https://user-images.githubusercontent.com/5152954/41219221-4a786ec4-6d7b-11e8-8be7-79d5582ee345.png)

But when I integrate autocomplete in the local demo, I do not get that meta information. I had also added the meta tag to my competitors.

![4](https://user-images.githubusercontent.com/5152954/41226756-14a1382a-6d90-11e8-94a7-a85161eaf529.png)

```
 editor.completers.push({
    getCompletions: function(editor, session, pos, prefix, callback) {
        callback(null, [
            { value: "foo", score: 100, meta: "local" },
            { value: "foobar", score: 100, meta: "keyword" }
        ]);
    }
});

```
 
Kindly guide me to resolve my issue.

**Comment by NaBian** [comment]
I had the same problem，In addition，It doesn't even appear in kitchen sink。
![demo](https://user-images.githubusercontent.com/17383395/43452518-3987ae2c-94ea-11e8-9957-42c93cdcf7bb.png)

**Comment by nightwing** [comment]
Thanks for the report. This issue should be fixed by #3730.

**Comment by muzichen** [comment]
![image](https://github.com/ajaxorg/ace/assets/7547416/09ec3dd9-1fc2-4b4a-81ae-a74c7d1c9f58)
![image](https://github.com/ajaxorg/ace/assets/7547416/2490586d-4989-4956-9482-738ecc9be339)
I had same problem, does anyone help me solve the problem.
